### PR TITLE
Substrate upgrade 0.9.38 to 0.9.40

### PR DESCRIPTION
**Pull request by FiberMan**
- [x] Upgrade Substrate version to `0.9.40`.
- [x] `subzero-dev` chainspec moved to `subzero`, so no need to support two almost identical folders. From now on we will handle differences between `subzero` and `subzero-dev` envs using different branches.
- [x] Add pallet_nfts.
- [x] bump tho 73

### CODE REVIEW

**Review by 2075 — APPROVED** [review]
lgtm

**Comment by 2075** [comment]
will you add contracts as well already?

**Comment by FiberMan** [comment]
> will you add contracts as well already?

contracts already there.

File: Cargo.toml [hunk 1]
```diff
@@ -7,12 +7,12 @@ members = [
 
 	# standalone network / dev environment:
 	# cargo build -p alphaville
-	"bin/alphaville/cli",
-	"bin/alphaville/runtime",
+	# "bin/alphaville/cli",
+	# "bin/alphaville/runtime",
 
 	# parachain dev environment
-	"bin/subzero-dev/node",
```
Comment (vayesy) [inline Cargo.toml:14]: `subzero-dev` & `subzero` should both be available here, please uncomment.

Comment (FiberMan) [reply Cargo.toml:14]: There will be only `subzero` available. We will build runtime for DEV using another branch that will contain dev changes (currently the only change for DEV is the presence of gamedao pallets).

Comment (vayesy) [reply Cargo.toml:14]: But `subzero-dev` is still built (just with different name).

**Review by vayesy — APPROVED** [review]

**Pull request merged**